index.html

```diff
@@ -84,30 +94,16 @@ <h3 >Soft Skills</h3>
                 </article>
             </div>
         </section>
-      
```

Comment:
you could remove everything currently inside the skills container and create the content dynamically. see code comments in index.js file.